### DISCUSSION TITLE
URL e DESTINATION agora podem ser também arquivos de texto!

### DISCUSSION THREAD

**Discussion by GabrielRF** [GabrielRF/Rss2Telegram#3] [Anúncios]
__Agora os valores de `URL` e `DESTINATION` podem ser definidos em arquivos de texto!__
Remova as duas variáveis dos _secrets_ e crie os arquivos `.txt` com os respectivos nomes. 
A ideia é facilitar a manutenção/alteração dos valores, pois estando em _secret_ não é possível ver o valor anterior.

__Atenção__: Caso a variável exista, o arquivo será ignorado!

**Comment by oCaioGuerras** [comment]
sera que é possivel criar de forma indivudual? Por exemplo, para rss X regra X, para rss Y, outras regras.

**Reply by GabrielRF** [reply]
@oCaioGuerras 
> sera que é possivel criar de forma indivudual?

Vou pensar em alguma forma de fazer isso.

Do jeito que está feito hoje acho que só tendo diferentes repositórios do Rss2Telegram, cada um com seu ajuste. 

---

__Pergunta__:
Vamos supor que tenha um jeito de organizar as regras por feed. Como você imagina que isso seria feito?
Você põe a url do feed em um arquivo e as regras no mesmo arquivo? Ou em um local separado?

Exemplo:
`feed1.txt` seria
```
https://blog.gabrf.com/feed.xml
ACCEPT:ALL
DROP:Política
```
Algo assim?
